### ISSUE TITLE
vgg16 has no dropout

### ISSUE THREAD

**Issue by don-tpanic** [keras-team/keras-applications#153]
I notice there is no dropout in the vgg16 code whereas dropout was used in the original paper. I wonder if there is a reason for removing it and whether the pretrained weights were learned with or without dropout?

Thank you

**Comment by don-tpanic** [comment]
Waiting..
